x-pack/plugins/security/public/components/authentication_state/_authentication_state.scss

```diff
@@ -1,22 +1,21 @@
-
-.loggedOut {
+.authenticationState {
```

Comment:
__note:__ that the only name I managed to find that would be somewhat related to both `logged_out` and `overwritten_session` :) Please share any other naming ideas that would work better!

Reply:
How about just changing `State` to `Page`? Then it could work for the login page as well. Also, these need a 3-letter prefix to conform to our BEM naming scheme. Can you add `sec` as the prefix to all of these? So this one would be: `.secAuthenticationPage`.

Reply:
> How about just changing State to Page? 

Hmm, I can change it to `...StatePage` since this is going to describe the finite "state of the authentication". Does that sound good?

> Then it could work for the login page as well. 

Not sure, I don't feel these have enough in common (logically) to generalize, the intention here is a bit different. I'd rather wait and see how it evolves.

> Also, these need a 3-letter prefix to conform to our BEM naming scheme. Can you add sec as the prefix to all of these?

Sure!

Reply:
> Not sure, I don't feel these have enough in common (logically) to generalize, the intention here is a bit different. I'd rather wait and see how it evolves.

Well the point of making this layout re-usable is more for the layout and visuals that come with it rather than for "purpose". So eventually, I'd like this to be surfaced even higher so that the login and welcome screens can use it as well so we minimize the copy/pasting of styles since that's all it's really applying here.

Reply:
Yeah, I definitely see your point and thanks for sharing your thoughts it's just that it should be a broader unification effort that's clearly out of scope of this PR IMO. This PR is already not a small/easy-to-review one.

> So eventually, I'd like this to be surfaced even higher so that the login and welcome screens can use it as well so we minimize the copy/pasting of styles since that's all it's really applying here.

That would be great, but will take some time :see_no_evil:

Reply:
Yeah I don't think that needs to happen now. Just wanted to respond to the worry about it not being general enough. When we do surface it up for those other pages we can rename the classes again so `...StatePage` is fine for now.